website/docs/core-concepts/writing-code/README.md

```diff
@@ -27,84 +50,31 @@ Appsmith currently supports two forms of JavaScript code for dynamically evaluat
    })()
 }}
 ```
-
-You can also write JavaScript code for event listeners. For JavaScript code inside an event listener, you can write multi-line JavaScript as below.
+* Verify the selected option and then perform operation on it:
 
 ```javascript
-{{
-  storeValue("userID", 42);  
-  console.log(appsmith.store.userID); 
-  showAlert("userID saved");
+{{ 
+  (function() {
+      if (Dropdown.selectedOptionValue === "1") {
+        return "Option 1";
+      } else {
+        return "Option 2";
+      }
+   })()
 }}
 ```
 
-## Configuring Widgets with Code
```

Comment:
this section should be moved to the subpage under Creating workflows titled 'Programming widgets
' as is for now so that we don't forget about it when working on that page

Reply:
This is part of Creating workflows. Added it there.

Reply:
Hey can we move it as part of this PR only as is so that when this is merged, we don't lose the section altogether. Creating workflows section might take a few days for review and publish

Reply:
I had already worked on it for the landing page, so I just moved it there. It should be available when you review the creating workflows page.